doc/uguide/absorption_theory.tex

```diff
@@ -2527,7 +2527,7 @@ \subsubsection{MPM93 water vapor absorption model}
 coefficients:
 \begin{equation}
   \label{eq:abs_theory:mpm93:absline}
-  \alphampmnl = 0.1820 \cdot \nu \cdot \phzo \cdot 
+  \alphampmnl = 0.1820 \cdot \nu \cdot
```

Comment:
First of all thanks a lot for your continued work on corrections for the theory guide. We really appreciate that (and are surprised how many typos there actually are ;-) ).
I have one question about the change in this line: Why did you remove the PH2O in this place? It seems to me it should be present here just like in all other MPM models?